control/depth_hold/src/gate_passing.py

```diff
@@ -0,0 +1,95 @@
+import rospy
```

Comment:
Why is files related to State machine located under control/depth_hold

Reply:
Have moved the files to a backup branch and removed them